lib/cog_cmd/cfn/stack/create.rb

```diff
@@ -1,62 +1,66 @@
-require_relative '../exceptions'
-require_relative '../helpers'
+require 'cog_cmd/cfn/helpers'
 
-class CogCmd::Cfn::Stack::Create < Cog::Command
+module CogCmd::Cfn::Stack
+  class Create < Cog::Command
 
-  include CogCmd::Cfn::Helpers
+    include CogCmd::Cfn::Helpers
 
-  USAGE = <<~END
-  Usage: cfn:stack create <stack name> <template name> [options]
+    attr_reader :stack_name, :template_name
+    attr_reader :stack_params, :tags, :policy, :notify, :on_failure, :timeout, :capabilities
 
-  Creates a new stack based on an existing template and returns the newly created stack.
+    def initialize
+      # args
+      @stack_name = request.args[0]
+      @template_name = request.args[1]
 
-  Options:
-    --param, -p "Key1=Value1"                   (Can be specified multiple times)
-    --tag, -t "Name1=Value1"                    (Can be specified multiple times)
-    --policy, -o "s3_policy_name"
-    --notify, -n "NotifyArn"                    (Can be specified multiple times)
-    --on-failure, -f <rollback | delete | keep>
-    --timeout, -e <minutes>
-    --capabilities, -c <iam | named_iam>
-  END
-
-  def run_command
-    unless stack_name = request.args[0]
-      raise CogCmd::Cfn::ArgumentError, "You must specify a stack name AND a template name."
+      # options
+      @stack_params = request.options['param']
+      @tags = request.options['tag']
+      @policy = request.options['policy']
+      @notify = request.options['notify']
+      @on_failure = request.options['on-failure']
+      @timeout = request.options['timeout']
+      @capabilities = request.options['capabilities']
     end
 
-    unless template_name = request.args[1]
-      raise CogCmd::Cfn::ArgumentError, "You must specify a stack name AND a template name."
+    def run_command
+      raise(CogCmd::Cfn::ArgumentError, "You must specify a stack name AND a template name.") unless stack_name
```

Comment:
I noticed in the next command you raise `Cog::Error`. Not sure which is right/better, but might be worth being consistent.